iphone/TitaniumKit/TitaniumKit/Sources/API/TiUtils.h

```diff
@@ -577,6 +577,8 @@ typedef enum {
 
 + (void)applyConstraintToView:(TiUIView *)view forProxy:(TiViewProxy *)proxy withBounds:(CGRect)bounds;
 
++ (NSString *)composeAccessibilityIdentifier:(TiUIView *)view;
```

Comment:
Can you change (TiUIView *) with (id) in above method declaration as TiUINavBarButton is not derived from TiUIView. So it is giving warning.

Reply:
done